substrate/frame/broker/src/lib.rs

```diff
@@ -61,7 +65,10 @@ pub mod pallet {
 	use sp_runtime::traits::{Convert, ConvertBack};
 	use sp_std::vec::Vec;
 
+	const STORAGE_VERSION: StorageVersion = StorageVersion::new(2);
```

Comment:
Was the bump to 1 not enough?

Reply:
Yeah, should be. Thanks for noticing this

Reply:
```suggestion
	const STORAGE_VERSION: StorageVersion = StorageVersion::new(1);
```

Reply:
No worries, I think I saw the try-runtime job flake at one stage a few days ago which I thought might have made you think you needed to bump again.

Reply:
Ah I've just seen it again - you need to add the broker migration to the runtimes' [migrations](https://github.com/paritytech/polkadot-sdk/blob/master/cumulus/parachains/runtimes/coretime/coretime-rococo/src/lib.rs#L109) (`coretime-rococo` migration CI job is failing)

Reply:
@Szegoo those two CI jobs are still failing, in case you didn't see this ^

Reply:
I somehow missed that comment. Updated the migrations now.